packages/forms/test/form_array_spec.ts

```diff
@@ -64,6 +64,91 @@ describe('FormArray', () => {
 
       expect(a.controls).toEqual([c1, c2, c3]);
     });
+
+    it('should not emit event when pushing if emitEvent is false', fakeAsync(() => {
+         a.valueChanges.subscribe((value) => {
+           throw 'Should not happen';
+         });
```

Comment:
I've looked at the code of this spec file and it seems like it's more common to log events and check the log array output:
```suggestion
         a.valueChanges.subscribe(() => logger.push('valueChanges'));
```
It's pretty minor, but would be great to have tests in a bit more consistent state.